go.mod

```diff
@@ -62,4 +62,5 @@ require (
 	google.golang.org/grpc v1.58.3 // indirect
 	google.golang.org/protobuf v1.31.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
+	sigs.k8s.io/yaml v1.4.0
```

Comment:
We already take a dependency on a YAML unmarshaller that observes `json` field tags: `github.com/ghodss/yaml`